sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -20,36 +106,21 @@ public enum ServiceVersion
             V1_0 = 1,
         }
     }
-    public partial class ContainerRepositoryClient
+    public partial class ContainerRepository
```

Comment:
Given our naming conventions and drawing from Storage, shouldn't this end in "Client" because, judging by the methods (including `GetProperties` as commented on above), this is a client.

And because it's a client, shouldn't it be constructable as you had it?

Reply:
We have this pattern in Service Bus - helper types that interact with the service, but are not suffixed with Client and cannot be constructed directly.

Reply:
I will say that it is a little more clear in Service Bus that these types would perform actions - the names ServiceBusSender/ServiceBusReceiver/ServiceBusProcessor all provide a hint that they do something.

Reply:
See comment above.  Given that this is not a client, I changed it so that it is not constructable, but must be retrieved from the official client which is the starting point for the SDK.

Reply:
Thanks @JoshLove-msft, it's great to know there's another precedent besides Cosmos here.  Krzysztof had also mentioned `Operation\<T\>` and `Pageable\<T\>` has examples of this type of helper class.